R/captions.R

```diff
@@ -21,18 +21,18 @@ captions <- list(
   ),
   demography = list(
     parameterSection = function(sectionId, parameterName) {
-      c(anchor(sectionId), "", paste("##", parameterName, "distributions"))
+      paste("##", parameterName, "distributions", anchor(sectionId))
     },
     xParameterSection = function(sectionId, parameterName) {
-      c(anchor(sectionId), "", paste("## ", parameterName, "-dependence", sep = ""))
+      paste("## ", parameterName, "-dependence ", anchor(sectionId), sep = "")
```

Comment:
let's put a blank between the title and the anchor everywhere, like it is done here

Reply:
There is already a blank which is included in the default arguments of the function `paste(..., sep = " ")` but this can be confusing as the function `paste0(..., sep = "")` uses a default with space